test/src/network.spec.ts

```diff
@@ -599,16 +599,28 @@ describe('network', function () {
 
       const events: string[] = [];
       page.on('request', request => {
-        return events.push(`${request.method()} ${request.url()}`);
+        // Ignore default favicon requests.
+        if (!request.url().endsWith('favicon.ico')) {
```

Comment:
let's use the isFavicon helper

Reply:
Oh, right, that's pretty handy, updated ✅